docs/10-input.md

```diff
@@ -0,0 +1,337 @@
+
+One real-world example is dragging and gestures. Sometimes a player will drag their
+finger outside your game and then into it, expecting it to work. If [[PointerScope]]
+is set to [[PointerScope.Canvas|Canvas]] this will not work. If it is set to
+[[PointerScope.Document|Document]], it will.
+
+### Responding to input
+
+The primary pointer can be a mouse, stylus, or single finger touch event. You
+can inspect what type of pointer it is from the [[PointerEvent]] handled.
+
+```js
+engine.input.pointers.primary.on('down', function(pe) {
+  if (pe.pointerType === ex.Input.PointerType.Mouse) {
+    ex.Logger.getInstance().info('Mouse event:', pe);
+  } else if (pe.pointerType === ex.Input.PointerType.Touch) {
+    ex.Logger.getInstance().info('Touch event:', pe);
+  }
+});
+```
+
+### Multiple Pointers (Multi-Touch)
+
+When there is more than one pointer detected on the screen,
+this is considered multi-touch. For example, pressing one finger,
+then another, will create two pointers. If you lift a finger,
+the first one remains and the second one disappears.
+
+You can handle multi-touch by subscribing to however many pointers
+you would like to support. If a pointer doesn't yet exist, it will
+be created. You do not need to check if a pointer exists. If it does
+exist, it will propagate events, otherwise it will remain idle.
+
+Excalibur does not impose a limit to the amount of pointers you can
+subscribe to, so by all means, support all 10 fingers.
+
+_Note:_ There is no way to identify touches after they happen; you can only
+know that there are _n_ touches on the screen at once.
+
+```js
+function paint(color) {
+  // create a handler for the event
+  return function(pe) {
+    if (pe.pointerType === ex.Input.PointerType.Touch) {
+      engine.canvas.fillStyle = color;
+      engine.canvas.fillRect(pe.x, pe.y, 5, 5);
+    }
+  };
+}
+engine.input.pointers.at(0).on('move', paint('blue')); // 1st finger
+engine.input.pointers.at(1).on('move', paint('red')); // 2nd finger
+engine.input.pointers.at(2).on('move', paint('green')); // 3rd finger
+```
+
+### Actor pointer events
+
+By default, [[Actor|Actors]] do not participate in pointer events. In other
+words, when you "click" an Actor, it will not throw an event **for that Actor**,
+only a generic pointer event for the game. This is to keep performance
+high and allow actors to "opt-in" to handling pointer events. Actors will automatically
+opt-in if a pointer related event handler is set on them `actor.on("pointerdown", () => {})` for example.
+
+To opt-in manually, set [[Actor.enableCapturePointer]] to `true` and the [[Actor]] will
+start publishing `pointerup` and `pointerdown` events. `pointermove` events
+will not be published by default due to performance implications. If you want
+an actor to receive move events, set [[CapturePointerConfig.captureMoveEvents]] to
+`true`.
+
+Actor pointer events will be prefixed with `pointer`.
+
+```js
+var player = new ex.Actor();
```

Comment:
There are many references to `var`, but `let` and `const` are recommended over it these days, because of their more logically defined scoping rules.